doc/async_worker.md

```diff
@@ -232,25 +229,25 @@ virtual ~AsyncWorker();
 operator napi_async_work() const;
 ```
 
-Returns the N-API napi_async_work wrapped by the AsyncWorker object. This can be
-used to mix usage of the C N-API and node-addon-api.
+Returns the N-API napi_async_work wrapped by the `Napi::AsyncWorker` object. This
+can be used to mix usage of the C N-API and node-addon-api.
 
 ## Example
 
-The first step to use the `AsyncWorker` class is to create a new class that inherit
-from it and implement the `Execute` abstract method. Typically input to your
+The first step to use the `Napi::AsyncWorker` class is to create a new class that
+inherits from it and implement the `Execute` abstract method. Typically input to your
```

Comment:
nit: full namespacing for &#96;Execute&#96;

Reply:
Done